x-pack/docs/en/rest-api/security/query-api-key.asciidoc

```diff
@@ -0,0 +1,258 @@
+[role="xpack"]
+[[security-api-query-api-key]]
+=== Query API key information API
+++++
+<titleabbrev>Query API key information</titleabbrev>
+++++
+
+Retrieves information for API keys with <<query-dsl,Query DSL>>
+in a <<paginate-search-results,pagniated>> fashion.
+
+[[security-api-query-api-key-request]]
+==== {api-request-title}
+
+`GET /_security/_query/api_key`
+
+`POST /_security/_query/api_key`
+
+[[security-api-query-api-key-prereqs]]
+==== {api-prereq-title}
+
+* To use this API, you must have at least the `manage_own_api_key` cluster privilege.
+
+[[security-api-query-api-key-desc]]
+==== {api-description-title}
+
+The information for the API keys created by
+<<security-api-create-api-key,create API Key>> can be retrieved using this API in
+a paginated manner and optionally filtered with a query.
+
+[[security-api-query-api-key-request-body]]
+==== {api-path-body-title}
+
+The following parameters can be specified in the body of a request and
+pertain to retrieving api keys:
+
+`query`::
+(Optional, string) A <<query-dsl,query>> to filter API keys to be returned.
+Note it only support a subset of the query types including
+<<query-dsl-match-all-query>>, <<query-dsl-bool-query>>,
+<<query-dsl-term-query>>, <<query-dsl-terms-query>>, <<query-dsl-ids-query>>,
+<<query-dsl-prefix-query>>, <<query-dsl-wildcard-query>>, <<query-dsl-range-query>>.
```

Comment:
Should we have some comments about the fields that can be queried? Maybe a list, or at least an explanation of how people would work out what the fields are.

Reply:
Thanks. I added a list to explain the available fields.